lib/ansible/modules/command.py

```diff
@@ -299,7 +299,7 @@ def main():
 
         try:
             os.chdir(chdir)
-        except (IOError, OSError) as e:
+        except OSError as e:
             r['msg'] = 'Unable to change directory before execution: %s' % to_text(e)
             module.fail_json(**r)
```

Comment:
This could be updated to take advantage of the improved error handling:

```suggestion
            r['msg'] = 'Unable to change directory before execution.'
            module.fail_json(**r, exception=e)
```